tools/rna_tools/sortmerna/sortmerna.xml

```diff
@@ -131,6 +165,56 @@ sortmerna --version 2>&1|grep 'SortMeRNA version'
         <param name="log" type="boolean" checked="False" truevalue="--log" falsevalue="" label="Generate statistics file"
                help="Generates statistics for the rRNA content of reads, as well as rRNA subunit distribution. (--log)">
         </param>
+	<conditional name="report">
+            <param name="report_type" type="select" display="radio" label="Parameters for filtering and read mapping" help="">
+                <option value="best" selected="true">Report best alignments per read reaching E-value</option>
+                <option value="num_alignments">Report first alignements per read reaching E-value</option>
+            </param>
+            <when value="best">
+                <conditional name="report_best">
+                    <param name="report_best_type" type="select" display="radio" label="Number of searched alignments" help="Only the best alignment is reported">
```

Comment:
Here I'm slightly more in favour of a select box. I like unified user interfaces and from other tools I would expect a normal drop down box and not radio buttons.